registry/proxy/proxyregistry.go

```diff
@@ -211,6 +211,15 @@ func (pr *proxyingRegistry) BlobStatter() distribution.BlobStatter {
 	return pr.embedded.BlobStatter()
 }
 
+type Closable interface {
```

Comment:
Let's rename this to `Closer` so it's more idiomatic `Go`. I know `proxy.Close()` doesn't need to return `error` but 

I usually write `Closer` interfaces so they're consistent wither other stdlib interfaces like `io.Closer` which has `Close()` returning `error`. Note: I wonder if `scheduler.Stop` should return error and if it does what's the blast radius of it

Reply:
Thanks for the review! The feedback is addressed.